tests/system/aiplatform/test_featurestore.py

```diff
@@ -79,7 +76,7 @@ def test_create_get_list_featurestore(self, shared_state):
         assert featurestore.resource_name == get_featurestore.resource_name
 
         list_featurestores = aiplatform.Featurestore.list()
-        assert (len(list_featurestores) - base_list_featurestores) == 1
+        assert len(list_featurestores) >= 1
```

Comment:
For my system test, I actually do:

```
        assert get_index.resource_name in [
            index.resource_name for index in list_indexes
        ]
```

Just another option that doesn't depend on the absolute number of items in the list.